web/Controllers/AdminController.cs

```diff
@@ -14,25 +14,33 @@ namespace BricksAndHearts.Controllers;
 public class AdminController : AbstractController
 {
     private readonly IAdminService _adminService;
+    private readonly ICsvImportService _csvImportService;
     private readonly ILandlordService _landlordService;
-    private readonly IPropertyService _propertyService;
     private readonly ILogger<AdminController> _logger;
+    private readonly IPropertyService _propertyService;
 
-    public AdminController(ILogger<AdminController> logger, IAdminService adminService, ILandlordService landlordService, IPropertyService propertyService)
+    public AdminController(ILogger<AdminController> logger, IAdminService adminService,
+        ILandlordService landlordService, IPropertyService propertyService, ICsvImportService csvImportService)
     {
         _logger = logger;
         _adminService = adminService;
         _landlordService = landlordService;
         _propertyService = propertyService;
+        _csvImportService = csvImportService;
     }
 
     [HttpGet]
     [Route("/admin")]
     public IActionResult AdminDashboard()
     {
         var isAuthenticated = User.Identity?.IsAuthenticated ?? false;
-        var viewModel = new AdminDashboardViewModel(_landlordService.CountLandlords(), _propertyService.CountProperties());
-        if (isAuthenticated) viewModel.CurrentUser = GetCurrentUser();
+        var viewModel =
+            new AdminDashboardViewModel(_landlordService.CountLandlords(), _propertyService.CountProperties());
+        if (isAuthenticated)
+        {
+            viewModel.CurrentUser = GetCurrentUser();
+        }

```

Comment:
I personally prefer the older version without the brackets but I'm not sure which (if any) of these are "better". I guess both have upsides and downsides :)

Reply:
I have absolutely no memory of changing that, I think maybe Rider just automatically reformatted it? That is to say, I don't have any strong feelings about this, so could easily change it back if you prefer.

Reply:
Dont worry about it, its probably more effort than its worth :) I just feel like I had nothing else worthwhile to comment on cause the rest of your code looked good!